### DISCUSSION TITLE
New Update

### DISCUSSION THREAD

**Discussion by brnwlshubh** [anilbeesetti/nextplayer#1014] [General]
@anilbeesetti May I know the time when the new update is going to release?

As Last update was release around 3 month ago.

**Comment by anilbeesetti** [comment]
Hey @brnwlshubh, I'm planning on releasing a new pre release version in a couple of weeks

**Reply by dairnatus** [reply]
Hello. Are there any plans to play videos in the background?

**Reply by anilbeesetti** [reply]
Yes, definitely but not in the upcoming release. I will try to get it ready for the next release

**Reply by Pentaphon** [reply]
>Are there any plans to play videos in the background?

I am waiting for this the most. Newpipe does this the best with Youtube videos and I hope Next Player can do it just as well with all videos.
